app/app.js

```diff
@@ -30,5 +30,7 @@ module.exports = function(args) {
   });
 
   app.post('/key', utils.promiseWrapper(krs.provisionKey));
+
+  app.post('/isUserKey', utils.promiseWrapper(krs.isUserKey));
```

Comment:
Do we want to do this in every case? Or is this strictly a BitGo + DAI requirement?

ie: If there is another KRS provider, would this still be relevant?

Reply:
Not sure about yours/others specific use cases and how you communicate with your KRS.

For us:

I think it becomes relevant once there IS another KRS provider. Eventually, I imagine this will turn into `verifyKey`, which would do the actual digest check against the key.

Currently, BitGo talks to our KRS, and our consumers talk to our Portal, hence the need (they're decoupled). Going forward, we will likely get BitGo to talk to the Portal as well (once it's up), and this will negate the need for the call as we'll be able to capture that it's one of our wallets on creation.

This is a good small agile step though, and easy to take out once/if redundant.

However, happy to keep the fork open on our end and port your changes whenever updates occur.